org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/participants/XMLSchemaErrorCode.java

```diff
@@ -176,37 +176,35 @@ public static Range toLSPRange(XMLLocator location, XMLSchemaErrorCode code, Obj
 			}
 			return XMLPositionUtility.selectTrimmedText(offset, document);
 		}
-		case SchemaLocation:
+		case SchemaLocation: { //xml xsi:schemaLocation

```

Comment:
It's  annoying because we have no test for this usecase. It should be niceto write a test for this usecase. https://github.com/apache/xerces2-j/blob/cf0c517a41b31b0242b96ab1af9627a3ab07fcd2/src/org/apache/xerces/impl/msg/XMLSchemaMessages.properties#L309 should help us to understand how to write test.

Reply:
Ok to test this usecase, please write a test with:

```xml
<ns:root  
  xmlns:ns="http://foo"  
  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"  
  xsi:schemaLocation="http://foo foo.xsd http://bar">
</ns:root>  
```